FitVid/src/Pages/Authentication/authentication.css

```diff
@@ -0,0 +1,162 @@
+.authentication{
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    /* background-color: var(--light-blue); */
+    border-radius: 8px;
+    width: 90%;
+    max-width: 93.7rem;
+    margin: 0rem auto;
+    padding: 5rem 1rem;
+    box-shadow: var(--primary-color) 0px 3px 8px;
+    gap: 1rem;
+    background-color: var(--background-color-dark);
+}
+.wrapper{
+    position: relative;
+    height: 100%;  
+}
+.blob{
+    position: absolute;
+    width: 30%;
+    z-index: -1;
+}
+.blob1{
+    top: 0px;
+    left: -100px;
+}
+.blob2{
+    bottom: 0px;
+    right: 0px;
+}
+.remember-me-checkbox{
+    margin-left: 4px;
+}
+.authentication-image-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    grid-column: 1 / 2;
+    z-index: 2;
+    width: 100%; 
+}
+.authentication-image-container img{
+    width: 100%;
+    max-width: 40rem;
+}
+.authentication-form-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    grid-column: 2 / 3;
+    z-index: 2;
+}
+.authentication-form{
+    background-color: var(--background-color);
+    max-width: 31.2rem;
+    width: 100%;
+    border-radius: 6px;
+    padding: 1rem 0.5rem;
+    box-shadow: var(--primary-color) 0px 3px 6px, var(--primary-color) 0px 3px 6px;
+}
+.authentication-form-heading{
+    text-align: center;
+    margin: 0 1rem 0.3rem 1rem;
+}
+.authentication-form-sub-heading{
+    margin: 0.3rem 1rem;
+    text-align: center;
+}
+.form-input-box{
+    position: relative;
+    display: flex;
+    justify-content: center;
+    margin: 1.8rem 0rem;
+}
+.form-input-box input{
+    width: 90%;
+    padding-left: 1rem;
+    font-size: 1rem;
+    height: 3.12rem;
+    box-shadow: rgba(0, 0, 0, 0.1) 0px 4px 12px;
+    border-radius: 3px;
+    border: 2px solid #BCBCBC;
+}
+.form-input-box input:focus{
+    outline: none;
+    border: 2px solid var(--primary-color);
+}
+.form-input-box i{
+    position: absolute;
+    top: 0.68rem;
+    right: 2.81rem;
+    cursor: pointer;
+    font-size: 1.5rem;
+}
+.mail-icon{
+    position: absolute;
+    top: 0.80rem;
+    right: 2.81rem;
+    cursor: pointer;
+    font-size: 1.5rem;
+    color: var(--color-black);
+}
+.checkbox-container{
+    width: 90%;
+    display: flex;
+    justify-content: space-between;
+    margin: 0 auto 1.8rem auto;
+}
+.forgot-password-btn{
+    color: var(--primary-text-color);
+    background-color: transparent;
+    cursor: pointer;
+}
+.forgot-password-btn:hover{
+    color: var(--primary-color);
+}
+.form-cta-button{
+    display: block;
+    width: 90%;
+    margin: 1rem auto;
+    border-radius: 3px;
+    text-align: center;
+}
+.login{
+    padding: 0;
+}
+.forgot-password{
```

Comment:
could be written as .login,.forgot-password{  padding : 0 } as they have same properties.